tyrant/cogs/magic_8_ball.py

```diff
@@ -0,0 +1,66 @@
+import random
+
+from discord.ext import commands
+from discord.ext.commands import Bot, Cog, Context
+
+from tyrant import constants
+
+# unsuitable responses to be removed from self.RESPONSES
+UNSUITABLE_RESPONSES = [
+    "I'm sorry Dave, I'm afraid I can't do that.",
+    "Not gonna happen.",
+    "Out of the question.",
+    "Can do!",
+    "You're the boss!",
+    "I got you.",
+    "No problem.",
+    "You got it!",
+]
+
+
+class Magic8Ball(Cog):
+    """Provide answers for generic yes or no questions."""
+
+    def __init__(self, bot: Bot):
+        """Initialize this cog with the Bot instance."""
+        self.bot = bot
+
+        self.negative_replies = list(constants.NEGATIVE_REPLIES)
+        self.positive_replies = list(constants.POSITIVE_REPLIES)
+        self.uncertain_replies = list(constants.UNCERTAIN_REPLIES)
+
+        # filtering responses
+        for response in UNSUITABLE_RESPONSES:
+            if response in self.negative_replies:
+                self.negative_replies.remove(response)
+            elif response in self.positive_replies:
+                self.positive_replies.remove(response)
+            elif response in self.uncertain_replies:
+                self.uncertain_replies.remove(response)
+
+    @commands.command(aliases=("8b", "8ball", "pleasesir"))
+    async def ask(self, ctx: Context, *, question: str = None):
+        """Send random answers to generic yes or no question."""
+        if question is None:
+            await ctx.send(
+                f'**The Tyrant proclaims **, "You are bad at asking questions"'
```

Comment:
```suggestion
                f'**The Tyrant proclaims**, "You are bad at asking questions"'
```
An evil space has snuck in here. 👾